composer.json

```diff
@@ -73,6 +73,7 @@
     "symfony/webpack-encore-bundle": "^2.0",
     "symfony/yaml": "^7.1",
     "twig/extra-bundle": "^2.12|^3.0",
+    "twig/intl-extra": "^3.20",
```

Comment:
what's that?

Reply:
It enables the following `{{ book.language|locale_name }}` which transforms `fr` into `French` (when locale is english)

Reply:
BTW, this fixes #394